### ISSUE TITLE
Expo Issue

### ISSUE THREAD

**Issue by khalilsarabey** [react-native-async-storage/async-storage#858]
### What happened?

Hello!
So I've been trying to use this library with a project built with expo, but I keep getting this error: 

[Unhandled promise rejection: Error: XMLHttpRequest failed: "this.getAsyncStorage is not a function. (In 'this.getAsyncStorage()', 'this.getAsyncStorage' is undefined)"]

![Screenshot 2022-11-03 at 2 07 11 AM](https://user-images.githubusercontent.com/4461476/199624902-ebfb848c-750c-49b2-a5a6-f977c7fd0f62.png)

And here's the error in the console:

![Screenshot 2022-11-03 at 2 07 48 AM](https://user-images.githubusercontent.com/4461476/199624979-cbe21bae-5da9-4cab-91c2-74f6da189334.png)

The query I'm trying to run is pretty straight forward

![Screenshot 2022-11-03 at 2 08 19 AM](https://user-images.githubusercontent.com/4461476/199625034-b4ed76a0-492e-4650-83ea-96b65f07e7b8.png)

Any idea what could be the issue? or am I doing something wrong?

Thanks!

### Version

1.17.3

### What platforms are you seeing this issue on?

- [ ] Android
- [X] iOS
- [ ] macOS
- [ ] Windows
- [ ] web

### System Information

```true
System:
    OS: macOS 13.0
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
    Memory: 1.99 GB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.18.0 - /usr/local/bin/node
    Yarn: Not Found
    npm: 8.19.2 - /usr/local/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.11.2 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 16.0, macOS 12.3, tvOS 16.0, watchOS 9.0
    Android SDK:
      API Levels: 28, 29, 30
      Build Tools: 28.0.3, 29.0.2, 30.0.2, 30.0.3
      System Images: android-29 | Google APIs Intel x86 Atom, android-30 | Google Play Intel x86 Atom, android-31 | Google Play Intel x86 Atom_64
      Android NDK: Not Found
  IDEs:
    Android Studio: 2021.1 AI-211.7628.21.2111.8139111
    Xcode: 14.0.1/14A400 - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_301 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.0.0 => 18.0.0 
    react-native: 0.69.6 => 0.69.6 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
info React Native v0.70.4 is now available (your project is running on v0.69.6).
info Changelog: https://github.com/facebook/react-native/releases/tag/v0.70.4.
info Diff: https://react-native-community.github.io/upgrade-helper/?from=0.69.6.
info To upgrade, run "react-native upgrade".
```

### Steps to Reproduce

- Initialize a new expo project
- add the library and parse sdk
- try to run a query

**Comment by khalilsarabey** [comment]
For anyone facing this issue, this got fixed in react 3.5.0-alpha.7, using the alpha7 version solves it.

**Comment by pluginslab** [comment]
Thanks for finding this out @khalilsarabey, what exactly do you mean when pointing to 3.5.0-alpha.7, you said react, did you mean parse? 🙏🏻

**Comment by pluginslab** [comment]
Figured out, this fix was not included in the 3.5.0 release, it is inside [3.5.1-beta1](https://github.com/parse-community/Parse-SDK-JS/pull/1587#issuecomment-1302584472).
